language/move-lang/src/expansion/translate.rs

```diff
@@ -799,10 +809,7 @@ fn spec(context: &mut Context, sp!(loc, pspec): P::SpecBlock) -> E::SpecBlock {
     sp(loc, E::SpecBlock_ { target, members })
 }
 
-fn spec_member(
-    context: &mut Context,
-    sp!(loc, pm): P::SpecBlockMember,
-) -> Option<E::SpecBlockMember> {
+fn spec_member(context: &mut Context, sp!(loc, pm): P::SpecBlockMember) -> E::SpecBlockMember {
```

Comment:
Thanks for looking at this! I landed it because the costs of doing so were minimal (trivial to reverse any changes we don't like) and the costs of waiting were high (continuous merge conflicts).

I think for this specific case your argument makes sense, but in general I really appreciate the lint. The usual case for me is that there's some error-handling code that's long-dead, and the lint flags such code.

(Note that the lint only flags private functions -- generally contracts like the one you're describing make a lot of sense for public functions, but the case tends to be weaker for private ones.)

I think adding `#![allow(clippy::unnecessary_wraps)]` to these files would be fine: https://rust-lang.github.io/rust-clippy/master/#unnecessary_wraps

Would you like to prepare the PR?

Reply:
Thanks for pointing to the attribute, I will use it next time I come along this code. We already turned some lint checks off via `x`, I'm wondering whats our process to accept or reject certain lint checks. Probably none ;-) Which is Ok because nothing is perfect.

Reply:
Yeah if you feel strongly that a particular lint should just be disabled then just open a pr to add it to the list in `x.toml` and we can go from there.